rental.rb

```diff
@@ -1,17 +1,28 @@
 require_relative './book'
+require_relative './person'
 
 class Rental
-  attr_accessor :date, :person, :book, :rentals
+  attr_accessor :date
+  attr_reader :person, :book
 
   def initialize(date, person, book)
-    @rentals = []
     @date = date
 
     @person = person
-    person.rentals << self unless @person.rentals.include?(self)
+    @person.rentals << self unless @person.rentals.include?(self)
 
     @book = book
-    book.rentals << self unless @book.rentals.include?(self)
+    @book.rentals << self unless @book.rentals.include?(self)
+  end
+
+  def person=(person)
+    @person = person
+    @person.rentals << self unless @person.rentals.include?(self)
+  end
+
+  def book=(book)
```

Comment:
- [ ] Great job in making the rental work, however, there is an error after typing the date. Kindly fix it.

<img width="513" alt="Screenshot 2022-09-02 at 9 23 37 AM" src="https://user-images.githubusercontent.com/70269507/188072441-abd56e23-1134-4d50-b9ab-680d52f8bf41.png">